recipes/terraform-provider-azuread/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set provider = "azuread" %}
+{% set goname = "github.com/terraform-providers/terraform-provider-"+provider.lower() %}
+{% set version = "0.8.0" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+{% set x4_plugin = name + '_v' + version + "_x4" + (".exe" if win else "")%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder:  {{ pkg_src }}
+    url: https://{{ goname }}/archive/v{{ version }}.tar.gz
+    sha256: eec745858a48905b95ffce60cc3b760285b49b5da10911aa7f7ec4120a9f6d75
+
+build:
+  skip: True  # [osx]
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - GOBIN=$(go env GOBIN)  # [not win]
+    - go build -v -o $GOBIN/{{ x4_plugin }} .  # [not win]
+    - for /f "usebackq tokens=*" %%a in (`go env GOBIN`) do SET GOBIN= %%a  # [win]
+    - go build -v -o %GOBIN%\{{ x4_plugin }} .  # [win]
+
+requirements:
+  build:
+    - {{ compiler('go') }} >=1.13
+  run:
+    - terraform >=0.12
+  run_constrained:
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.13") }}  # [osx]
```

Comment:
```suggestion
```
this is not needed, it is pushed down as a strong [run_export][run_export] from the go compiler when running on OSX.

[run_export]: https://github.com/conda-forge/go-feedstock/blob/5ad419e63a613e2384809dcd040acbf6f251bf99/recipe/meta.yaml#L91-L93